os/tools/configure.sh

```diff
@@ -245,4 +245,45 @@ if [ "X${defappdir}" = "Xy" ]; then
     echo "CONFIG_APPS_DIR=\"$posappdir\"" >> "${dest_config}"
   fi
 fi
+
+# Check mandatory package to configuration set
+IOTIVITY=`grep CONFIG_ENABLE_IOTIVITY=y ${dest_config}`
+if [ ! -z ${IOTIVITY} ]; then
+  if ! which scons >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To build ${boardconfig}, scons should be installed"
+    echo "  Find details at https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseIotivity.md"
+    echo " "
+  fi
+fi
+
+ROMFS=`grep CONFIG_FS_ROMFS=y ${dest_config}`
+if [ ! -z ${ROMFS} ]; then
+  if ! which genromfs >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To download ${boardconfig} built image, genromfs should be installed"
+    echo "  Find details at https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseROMFS.md"
+    echo " "
+  fi
+fi
+
+IOTJS=`grep CONFIG_ENABLE_IOTJS=y ${dest_config}`
+if [ ! -z ${IOTJS} ]; then
+  if ! which cmake >/dev/null; then
+    echo " "
+    echo "  CAUTION!! To build ${boardconfig}, cmake should be installed"
```

Comment:
How about providing this guide - https://cmake.org/install/?

Reply:
There is no specific README of iotjs to build it on TizenRT. That's why I only miss it.
But you want to provide a guidance. Right? Then, how about a generic README of iotjs? But it does not show how to do it on TizenRT.
I don't agree adding specific package installlation on this script.

Reply:
Adding README for iotjs is good option. But if we don't have one then at least  it is better to point to generic cmake installation guide.

Reply:
I added iotjs README.